### PR TITLE
HParams: Clarify what happens when a new value is input to some methods

### PR DESCRIPTION
**Pull request by ThisIsPIRI**
The current docstrings don't state that the methods don't accept new hyperparameters.

### CODE REVIEW

**Review by ctrlaltvikas — APPROVED** [review]

**Comment by tensorflowbutler** [comment]
Nagging Reviewer : You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 14 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer : You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 29 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer : You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 44 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer : You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 59 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer : You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 74 days with no activity and the `awaiting review` label has been applied.

**Review by danabo — APPROVED** [review]
LGTM

**Comment by tensorflowbutler** [comment]
Nagging Assignee @drpngx: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @drpngx: It has been 29 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @drpngx: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by hgadig** [comment]
@ebrevdo  Could you please verify and LGTM this ?

File: tensorflow/contrib/training/python/training/hparam.py [hunk 1]
```diff
@@ -543,7 +544,7 @@ def parse(self, values):
     return self.override_from_dict(values_map)
 
   def override_from_dict(self, values_dict):
-    """Override hyperparameter values, parsing new values from a dictionary.
+    """Override existing hyperparameter values, parsing new values from a dictionary.
```
Comment (ebrevdo) [inline tensorflow/contrib/training/python/training/hparam.py]: this line is over 80 chars long; please shorten.
